Demo/Demo/FeatureViewControllers/CardDemoViewController.swift

```diff
@@ -125,7 +125,8 @@ class CardDemoViewController: FeatureBaseViewController, UITextFieldDelegate {
         let config = CoreConfig(clientID: DemoSettings.clientID, environment: DemoSettings.environment.paypalSDKEnvironment)
         let cardClient = CardClient(config: config)
 
-        cardClient.approveOrder(orderID: orderID, card: card) { result in
+        let cardRequest = CardRequest(orderID: orderID, card: card)
+        cardClient.approveOrder(request: cardRequest) { result in
```

Comment:
Just a thought based on the swift docs you sent over yesterday, but we _could_ name this function externally like is used here so that it reads as:
```
cardClient.approveOrder(with: cardRequest) { result in
```
Not a blocker by any means, but just a thought to potentially improve merchant clarity of this function. 😄

Reply:
Def! Yeah that does read nicely.